src/ast/codegen_llvm.cpp

```diff
@@ -1286,6 +1286,8 @@ void CodegenLLVM::visit(Probe &probe)
      * can restore it for the next pass (printf_id_, time_id_).
      */
     int starting_printf_id_ = printf_id_;
+    int starting_cat_id_ = cat_id_;
+    int starting_system_id_ = system_id_;
     int starting_time_id_ = time_id_;
     int starting_join_id_ = join_id_;
```

Comment:
Can you change these local variables to not have the trailing `_`?

Was kinda confusing reading this.